focuspoints.lrdevplugin/Utils.lua

```diff
@@ -68,7 +68,35 @@ function split(str, delim)
 end
 
 --[[
- Splits a string into 2 parts: key and value. 
+-- Split a key/value table and returns 2 strings being the ordered list of keys and the ordered list of value
+-- as well as the max key length, the max value length and the number of lines in both strings
+-- tab - the table
+--]]
+function splitForColumns(tab)
+  local keys = ""
+  local values = ""
+  local keys_max_length = 0
```

Comment:
I canot name it table as I use table.insert after, if this is what you mean.
We can/should rename table to tab in the other methods thus...